crates/schema/src/def/validate/v9.rs

```diff
@@ -582,6 +601,12 @@ impl TableValidator<'_, '_> {
         }
     }
 
+    fn validate_row_level_security_def(&mut self, policy: RawRowLevelSecurityDefV9) -> Result<RowLevelSecurityDef> {
+        let RawRowLevelSecurityDefV9 { name, sql } = policy;
```

Comment:
It would be nice to also parse the sql here as well and validate it. Maybe add a TODO?

Reply:
This is for a follow-up pr. We are still waiting to integrate the new parser...

Reply:
Where is this validation utility called? Is this called by the rust bindings for example? If so, we can't do any validation here, because currently semantic analysis can only be run against the server.

Reply:
@mamcx ah yeah I meant later, for now just a TODO is fine.

Reply:
@joshua-spacetime this is called while parsing the `RawModuleDef` that comes out of a module, both in the server while instantiating a module and in, say, the CLI, while performing tasks like generating client bindings.